config.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
     secret: process.env.ICASEWORK_SECRET,
     type: process.env.ICASEWORK_FORM_TYPE || '',
     db: process.env.ICASEWORK_DB,
-    timeout: +process.env.ICASEWORK_TIMEOUT || 60000
+    timeout: +process.env.ICASEWORK_TIMEOUT || 60000,
+    fetchTimeout: +process.env.ICASEWORK_FETCH_TIMEOUT || 30000
```

Comment:
adding a shorter timeout in case of network issues but this is completely arbitrary